src/main/java/com/github/dockerjava/api/command/InspectContainerResponse.java

```diff
@@ -134,7 +139,12 @@ public String getResolvConfPath() {
         return volumes == null ? null : volumes.getBinds();
     }
 
+    /**
+     * @deprecated As of {@link RemoteApiVersion#VERSION_1_20}
+     * use {@link #getMounts()} instead
+     */
     @JsonIgnore
+    @Deprecated
```

Comment:
`VolumesRW` field is removed with 1.20. This may lead to `NullpointerException`s, so adding `@CheckForNull` now.